xdsl/parser/attribute_parser.py

```diff
@@ -184,24 +184,40 @@ def _parse_dialect_type_or_attribute_inner(
         The contents will be parsed by a user-defined parser, or by a generic parser
         if the dialect attribute/type is not registered.
         """
+        pretty = "." in attr_name
```

Comment:
Can you please update the comment in the docstring above based on what is actually parsed now?

Reply:
I would probably change the name of the function as well.
Probably having both a `parse_pretty_dialect_type_inner`, and a `parse_opaque_dialect_type_inner`?
That should make things clearer.

Reply:
You probably want a function like `_parrse_attribute_parameters` that will dispatch to the right function as well, that should make things clearer.

Reply:
Happy to do a small refactor!